internal/edot/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.24.10
 replace github.com/elastic/elastic-agent => ../../
 
 require (
-	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20251121190645-8580e1a4ad09
+	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20251125165534-04732ce4ee33
```

Comment:
This is pointed at a commit on main https://github.com/elastic/beats/commit/04732ce4ee3337a75dc26e672066fda7cc4fc333